website/src/pages/404.tsx

```diff
@@ -1,11 +1,13 @@
 import { Box, Button, Center, Link, Text, theme, useColorModeValue } from "@chakra-ui/react";
+import { useTranslation } from "next-i18next";
 import { AlertTriangle } from "lucide-react";
 import Head from "next/head";
 import { EmptyState } from "src/components/EmptyState";
 import { getTransparentHeaderLayout } from "src/components/Layout";
 export { getDefaultStaticProps as getStaticProps } from "src/lib/default_static_props";
 
 function Error() {
+  const { t } = useTranslation(["common"]);
```

Comment:
Thanks. If common is the only namespace you don't need to specify it because it's part of get default static props.

Reply:
Oh, thanks